src/sys/signal.rs

```diff
@@ -788,28 +784,6 @@ mod sigevent {
         }
     }
 
-    impl Debug for SigEvent {
```

Comment:
I think `SigEvent`'s manual `Debug` implementation is better than the derived one, because it won't print unused fields.

Reply:
I'm thinking this should be moved into `libc`, so I'll need to submit another PR to do that.

Reply:
Opened rust-lang/libc#1394 to move this there

Reply:
That PR's been merged so resolving.